rules/integrations/okta/credential_access_okta_authentication_for_multiple_users_from_single_source.toml

```diff
@@ -0,0 +1,125 @@
+[metadata]
+creation_date = "2024/06/17"
+integration = ["okta"]
+maturity = "production"
+min_stack_comments = "ES|QL rule type is still in technical preview as of 8.13, however this rule was tested successfully"
+min_stack_version = "8.13.0"
+updated_date = "2024/06/17"
```

Comment:
Asked the same thing in an Isai PR, as ES|QL is GA since 8.14.0, do we want to change anything related to the min_stack, or do we keep it like this?

Reply:
Updated.